docs/remote/containers.md

```diff
@@ -192,225 +172,7 @@ Note that if the container fails to come up due to something like a Docker build
-
-Let's set up a container for use with all of the Python projects in the `./Repos/python` folder.
-
-1. Start VS Code, select **Remote-Containers: Open Folder in Container...** from the Command Palette (`kbstyle(F1)`) or quick actions Status bar item, and select the `./Repos/python` folder.
-
-    > **Tip:** If you want to edit the container's contents or settings before opening the folder, you can run **Remote-Containers: Add Development Container Configuration Files...** instead.
-
-    ![Quick actions Status bar item](images/common/remote-dev-status-bar.png)
-
-2. Now pick a starting point for your dev container. You can either select a base **dev container definition** from a filterable list, or use an existing [Dockerfile](https://docs.docker.com/engine/reference/builder/) or [Docker Compose file](https://docs.docker.com/compose/compose-file/#compose-file-structure-and-examples) if one exists in the folder you selected.
-
-    > **Note:** When using Alpine Linux containers, some extensions may not work due to `glibc` dependencies in native code inside the extension.
-
-    ![Select a python dev container definition](images/containers/select-dev-container-def-python.png)
-
-    The list will be automatically sorted based on the contents of the folder you open. Note the dev container definitions displayed come from the [vscode-dev-containers repository](https://aka.ms/vscode-dev-containers). You can browse the `containers` folder of that repository to see the contents of each definition.
-
-3. After picking the starting point for your container, VS Code will add the dev container configuration files to the `./Repos/python/.devcontainer` folder.
-
-4. The VS Code window will reload and start building the dev container. A progress notification provides status updates. You only have to build a dev container the first time you open it; opening the folder after the first successful build will be much quicker.
-
-    ![Dev Container Progress Notification](images/containers/dev-container-progress.png)
-
-5. After the build completes, VS Code will automatically connect to the container. Once connected use **File > Open... / Open Folder...** to select one of the folders under `./Repos/python`.
-
-    ![Open python folder in the container](images/containers/open-folder-python.png)
-
-6. In a moment, VS Code will open the folder inside the same container. In the future, you can use the **Remote Explorer** in the Activity Bar to open this sub-folder in the container directly.
-
-    ![Container explorer with multiple folders under python container](images/containers/containers-explorer-python.png)
-
-> **Tip:** Instead of mounting the local filesystem, you can use a similar flow to set up a container with an isolated, more performant volume that you clone your source code into. See the [Advanced Containers](/docs/remote/containers-advanced.md#use-a-named-volume-for-your-entire-source-tree) article for details.
-
-## Creating a devcontainer.json file
```

Comment:
Reading through this, we really don't talk about devcontainer.json in this article until i starts to be referenced in later sections on extensions, etc. It feels a bit disjointed given its a central concept on first pass.

I wonder if this introduction could stay to help bridge the gap.  The "for example" part of this could then point to the other article.